packages/time/src/readable-semester.ts

```diff
@@ -0,0 +1,28 @@
+// @ts-check
+/* eslint-disable max-statements-per-line */
+
+/**
+ * Returns a struct with year and semester in readable format
+ */
+
+interface ReadableSemester {
+	year: string;
+	season: string;
+}
+
+export function readableSemester(semester: string): ReadableSemester {
```

Comment:
IMO we should move the semester class from Client to here. It has a lot more features that should cover most/all of the use case.

Reply:
That could work. The semester class in client already had this built in function, so moving it here would make sense.